components/button/Button.js

```diff
@@ -0,0 +1,30 @@
+export default class Button {
+  constructor(key, code, parent) {
+    this._key = key;
+    this._code = code;
+    this._parent = parent;
+
+    this._button = this._createButton();
+  }
+
+  changeButtonValue(newKey) {
+    this._button.innerHTML = newKey;
+    this._button.setAttribute('data-key', newKey);
+  }
+
+  setUpperCase() {
+    this._button.classList.add('clicked');
+  }
+
+  _createButton() {
+    const button = document.createElement('div');
```

Comment:
It's not so crucial but it would be better to use here 'button' element instead of div.